cmd/server/flags.go

```diff
@@ -459,18 +459,13 @@ var flags = append([]cli.Flag{
 	//
 	&cli.StringFlag{
 		EnvVars:  []string{"WOODPECKER_ENCRYPTION_KEY"},
-		Name:     "encryption-raw-key",
-		Usage:    "Raw encryption key",
+		Name:     "encryption-key",
+		Usage:    "AES encryption key",
 		FilePath: os.Getenv("WOODPECKER_ENCRYPTION_KEY_FILE"),
 	},
 	&cli.StringFlag{
 		EnvVars: []string{"WOODPECKER_ENCRYPTION_TINK_KEYSET_FILE"},
 		Name:    "encryption-tink-keyset",
 		Usage:   "Google tink AEAD-compatible keyset file to encrypt secrets in DB",
 	},
-	&cli.BoolFlag{
```

Comment:
there should be an option to revert encryption back to unencrypted ... we could add a warning if that's really the intended case ... but we should have it

Reply:
> make sure migration of non encrypted and back

Done

> mix in-between

This can be achieved via disabling encryption. For example, Plain -> AES -> Plain -> Tink. I would leave it like that, for this PR at least.